cmd/oras/blob/fetch.go

```diff
@@ -48,17 +49,23 @@ func fetchCmd() *cobra.Command {
 
 ** This command is in preview and under development. **
 
-Example - Fetch the blob and save it to a local file:
+Example - Fetch a blob from registry and save it to a local file:
   oras blob fetch --output blob.tar.gz localhost:5000/hello@sha256:9a201d228ebd966211f7d1131be19f152be428bd373a92071c71d8deaf83b3e5
 
-Example - Fetch the blob and print the raw blob content:
+Example - Fetch a blob from registry and print the raw blob content:
   oras blob fetch --output - localhost:5000/hello@sha256:9a201d228ebd966211f7d1131be19f152be428bd373a92071c71d8deaf83b3e5
 
 Example - Fetch and print the descriptor of a blob:
   oras blob fetch --descriptor localhost:5000/hello@sha256:9a201d228ebd966211f7d1131be19f152be428bd373a92071c71d8deaf83b3e5
 
-Example - Fetch the blob, save it to a local file and print the descriptor:
+Example - Fetch a blob, save it to a local file and print the descriptor:
   oras blob fetch --output blob.tar.gz --descriptor localhost:5000/hello@sha256:9a201d228ebd966211f7d1131be19f152be428bd373a92071c71d8deaf83b3e5
+
+Example - Fetch and print a blob from OCI image layout folder 'layout-dir':
```

Comment:
Is it exact to use `layout-dir@sha256:9a201d228ebd966211f7d1131be19f152be428bd373a92071c71d8deaf83b3e5` to represent an OCI layout folder?

Reply:
`layout-dir@sha256:9a201d228ebd966211f7d1131be19f152be428bd373a92071c71d8deaf83b3e5` means fetch a blob with digest `sha256:9a201d228ebd966211f7d1131be19f152be428bd373a92071c71d8deaf83b3e5` from an OCI image layout folder 'layout-dir'